src/platforms/hosted/meson.build

```diff
@@ -103,16 +108,23 @@ if build_machine.system() in ['windows', 'cygwin']
 		cc.find_library('setupapi'),
 	]
 	bmda_sources += files('serial_win.c')
+
+	if not cc.has_function('getopt_long', prefix: '#include <getopt.h>')
+		bmda_deps += [
+			libgetopt
+		]
+	endif
 else
 	libftdi = dependency('libftdi1', required: false, method: 'pkg-config', native: is_cross_build)
 	bmda_sources += files('serial_unix.c')
 endif
 
 # Pick the appropriate HIDAPI depending on platform
 if build_machine.system() == 'linux'
-	hidapi = dependency('hidapi-hidraw', required: false, method: 'pkg-config', native: is_cross_build)
+	hidapi = dependency('hidapi-hidraw', required: false, method: 'pkg-config', fallback: 'hidapi', native: is_cross_build)
```

Comment:
`required: false` here can be dropped given this always has the fallback now. The only reason for putting `required: false` in here at all was so BMDA not finding its deps wouldn't explode - hence why `libusb` having a fallback, also doesn't specify `required: false` as the aim is to get all deps to this state eventually.